src/main/resources/application.yaml

```diff
@@ -15,7 +15,7 @@ management:
 spring:
   jpa:
     database-platform: org.hibernate.dialect.PostgreSQLDialect
-    show-sql: false
+    show-sql: true
     properties:
       hibernate:
         jdbc:
```

Comment:
can this be set to false again after testing is completed?

Reply:
yep, it is no longer needed now, i will change it back